vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -131,6 +134,7 @@ type bucket struct {
 	userAgent      string
 	name           string
 	billingProject string
+	mu             sync.Mutex
```

Comment:
Is this being used somewhere?

Reply:
Actually it was being used when I was checking latency. But since I removed the latency code so I have removed this also.